spec/system/department_admin_should_be_able_to_manage_assemblies_spec.rb

```diff
@@ -58,7 +58,7 @@
       expect(page).to have_admin_callout("successfully")
       expect(Decidim::Assembly.last.area).to eq(area)
 
-      within ".container" do
+      within ".card#assemblies" do
         # expect(page).to have_current_path decidim_admin_assemblies.assemblies_path(q: { parent_id_eq: parent_assembly&.id })
```

Comment:
uncomment and check that it works for v0.28

Reply:
is not works...

Reply:
ok...